<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_ressources</class>
 <widget class="QDialog" name="Dialog_ressources">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edition des ressources</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../openjardin.qrc">:/images/ressources.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_idRessource">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>RESSOURCES</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_Ressource"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="btnMoyenEdit">
           <property name="maximumSize">
            <size>
             <width>26</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Editer un moyen</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../openjardin.qrc">
             <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_idMoyen">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_designation_moyen"/>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Coût</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEdit_Designation"/>
       </item>
       <item row="10" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </item>
       <item row="7" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_9"/>
       </item>
       <item row="6" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Prévu :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_quantitePrevue"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Réalisé :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_quantiteUtilisee"/>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Unité :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_unite">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_validerQuantiteMoyen">
           <property name="toolTip">
            <string>Valider la quantité du moyen sélectionné</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../openjardin.qrc">
             <normaloff>:/images/Valider.png</normaloff>:/images/Valider.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Quantité</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QTableView" name="tableView_moyens_alloues"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Moyens alloués</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolButton" name="toolButton_addMoyen">
           <property name="toolTip">
            <string>Ajouter un moyen</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../openjardin.qrc">
             <normaloff>:/images/add_48.png</normaloff>:/images/add_48.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_supprimeMoyen">
           <property name="toolTip">
            <string>Supprimer un moyen</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../openjardin.qrc">
             <normaloff>:/images/remove_48.png</normaloff>:/images/remove_48.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_idListeRessource">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Désignation</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Prévu</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_coutPrevu">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Total actualisé</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_coutTotal"/>
         </item>
        </layout>
       </item>
       <item row="2" column="2">
        <widget class="QTextEdit" name="textEdit_Commentaires"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Commentaires</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Tâche utilisatrice</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLineEdit" name="lineEdit_IdTache">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Designation_tache"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_nouveau">
         <property name="text">
          <string>Fiche vierge</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Supprimer">
         <property name="text">
          <string>Supprimer</string>
         </property>
         <property name="icon">
          <iconset resource="../openjardin.qrc">
           <normaloff>:/images/remove_48.png</normaloff>:/images/remove_48.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ajout_ressource">
         <property name="toolTip">
          <string>Ajouter la ressource créée à la tâche</string>
         </property>
         <property name="text">
          <string>Valider et Ajouter à la tâche</string>
         </property>
         <property name="icon">
          <iconset resource="../openjardin.qrc">
           <normaloff>:/images/Valider.png</normaloff>:/images/Valider.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Annuler">
         <property name="text">
          <string>Fermer</string>
         </property>
         <property name="icon">
          <iconset resource="../openjardin.qrc">
           <normaloff>:/images/icones/quitter.png</normaloff>:/images/icones/quitter.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../openjardin.qrc"/>
 </resources>
 <connections/>
</ui>
